<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndMain</class>
 <widget class="QMainWindow" name="wndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Data Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpDataSource">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Source</string>
    </property>
    <widget class="QRadioButton" name="rbtDataFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDataFile">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDataBrowse">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtDataSerial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSerialPort">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDataSerialNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDataSerialGet">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Get File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDataSerialList">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>List File</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFileNumber">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>49</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpDataCalib">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Calibration</string>
    </property>
    <widget class="QRadioButton" name="rbtCalibPre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Predefined</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtCalibCustom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>94</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbCalibPre">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>211</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCalibCustom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalibCustom">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpGraph">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>761</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Result Graph</string>
    </property>
    <widget class="QPushButton" name="btnDraw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QwtPlot" name="plotMain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>741</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
